src/OpenSpaceToolkit/Physics/Units/Derived/Angle.cpp

```diff
@@ -1,5 +1,5 @@
 /// Apache License 2.0
-
+#include <cmath>
```

Comment:
Simply for consistency:

```suggestion
#include <cmath>

```